modules/govuk_unattended_reboot/templates/usr/local/bin/with_reboot_lock.erb

```diff
@@ -19,4 +20,4 @@ function unlock() {
 lock
 trap "unlock" EXIT
 
-"${@}"
+"${@:222}"
```

Comment:
Rather than pulling a substring out of the args, I'd just shift out the first argument and proceed as usual
```suggestion
shift
"${@}"
```

Reply:
Haha, this was meant to be 2, rather than 222...

But yeah, I think this is fine as well.

Reply:
I've amended the original commit now to use shift rather than `"${@:2}"`